.dependabot/config.yml

```diff
@@ -0,0 +1,10 @@
+version: 1
+update_configs:
+  - package_manager: "python"
+    directory: "/"
+    update_schedule: "weekly"
+    default_labels:
+      - "dependencies"
+    default_reviewers:
+      - "Zlopez"
+      - "vfazio"
```

Comment:
@Zlopez, @vfazio, are you two alright with being default reviewers? I sourced you two from the [most recent contributors](https://github.com/release-monitoring/anitya/graphs/contributors?from=2019-05-21&to=2019-10-04&type=c) GitHub page.

Alternatively or additionally, do you have suggestions for who should be default reviewers?